packages/pricing/src/services/pricing-module.ts

```diff
@@ -94,13 +94,21 @@ export default class PricingModuleService<
     InjectedDependencies,
     PricingTypes.PriceSetDTO,
     {
-      MoneyAmount: { dto: PricingTypes.MoneyAmountDTO }
+      MoneyAmount: {
+        dto: PricingTypes.MoneyAmountDTO
```

Comment:
very-nit and non-blocking: I am not super fond of the naming here. 

I would prefer something like:
```
{
  common: PricingTypes.MoneyAmountDTO,
  create:  ...,
  update: ...,
}
```

Reply:
I can do the create and update change as a part of this pr, but the dto change would be huge 😄 , can do that as a different PR, i'm currently in merge hell rn.

Reply:
I believe that the suffix dto was more self describing than create which is an action verb. Also, we are missing the update overload no?

Reply:
done

Reply:
If you are configuring this type argument, I would presume users are aware that they are working with DTOs for the abstract service factory, which is why I think the `_dto` is redundant.

Reply:
As you prefer, i just think an action verb might be more confusing for the configuration. Maybe adding more documentation to it will help

Reply:
Maybe something to help would be to rename the template arg to TModelConfigDtos instead of model config

Reply:
Should we add the same shape for the main model dto config then?

Reply:
I see most modules override the create and update, so i'm leaning towards keep that out for now.